js/bundle/src/cli.ts

```diff
@@ -37,55 +44,91 @@ export function addFilesRecursively(
   }
 }
 
-export function main() {
-  const options = commander
+function readOptions() {
+  return commander
     .requiredOption('-d, --dir <directory>', 'input root directory (required)')
     .option('-b, --baseURL <URL>', 'base URL')
     .option(
       '-f, --formatVersion <formatVersion>',
-      'webbundle format version, possible values are "b1" and "b2" (default: "b2")',
-      'b2'
+      'webbundle format version, possible values are ' +
+        APPROVED_VERSIONS.map((v) => `"${v}"`).join(' and ') +
+        ' (default: "' +
+        DEFAULT_VERSION +
+        '")',
+      DEFAULT_VERSION
     )
     .option(
       '-p, --primaryURL <URL>',
-      'primary URL (defaults to base URL, only valid with format version "b1")'
+      'primary URL (defaults to base URL, only valid with format version "' +
+        B1 +
+        '")'
     )
     .option(
       '-m, --manifestURL <URL>',
-      'manifest URL (only valid with format version "b1")'
+      'manifest URL (only valid with format version "' + B1 + '")'
     )
     .option('-o, --output <file>', 'webbundle output file', 'out.wbn')
     .parse(process.argv);
+}
 
+function validateOptions(options: any) {
```

Comment:
Let's explicitly write the return type if it's not obvious from the function name.
```suggestion
function validateOptions(options: any): string | null {
```